core/src/it/scala/org/ensime/indexer/SearchServiceSpec.scala

```diff
@@ -255,19 +392,34 @@ object SearchServiceTestUtils {
     results
   }
 
-  def searchExpectEmpty(query: String)(implicit service: SearchService) = {
+  def searchExpectEmpty(query: String)(implicit service: SearchService, p: Position) = {
     val max = 1
     val results = service.searchClassesMethods(List(query), max)
-    withClue("expected empty results from %s".format(query))(results shouldBe empty)
+    withClue(s"expected empty results from $query")(results shouldBe empty)
     results
   }
 
-  def searchesEmpty(queries: String*)(implicit service: SearchService) =
+  def searchesEmpty(queries: String*)(implicit service: SearchService, p: Position) =
     queries.toList.foreach(searchExpectEmpty)
 
   // doesn't assert that expect finds itself because the lucene query
   // syntax conflicts with the characters in method FQNs
-  def searchesMethods(expect: String, queries: String*)(implicit service: SearchService) =
+  def searchesMethods(expect: String, queries: String*)(implicit service: SearchService, p: Position) =
     (queries.toList).foreach(searchClassesAndMethods(expect, _))
 
+  def getClassHierarchy(fqn: String, hierarchyType: Hierarchy.Direction)(implicit service: SearchService, p: Position): Hierarchy = {
```

Comment:
might make sense just to return this as a `Future` (but then you probably end up blocking somewhere else...)